### PR TITLE
Rework cube presentation

### PR DESCRIPTION
**Pull request by alexAubin**
Right now, the first explanation on the main page (apart from the introduction video) focuses on technical stuff related to the VPN

This is a proposal to recenter the presentation in more practical terms, about the two main functions of the internet cube : 
- cleaning the internet access to improve your privacy
- having your very own piece of internet where you can self-host your services

Additional work should be done later in the page to clarify that this does not mean that your internet is "more secure" (word typically used by people for whom what 'security' means is fuzzy) or that it's not the end of the story but just part of a general ecocystem : e.g. if you really want your whole computer usage to be more private, you should use Linux, LineageOS, use services from Framasoft/CHATONS if you cannot host them, use TOR if you need privacy browsing the internet, explain that having a static IP might be the opposite of privacy in some sense, etc...

### CODE REVIEW

File: en.html [hunk 1]
```diff
@@ -32,63 +32,60 @@ <h1><img src="img/cube.png" alt="The Internet Cube" ></h1>
...
+              <li><span>your server is connected to an internet access that respect Net Neutrality and which isn't censored</span>, with static IPv4 and IPv6 and there is no need to configure your ISP box;</li>
+              <li>if you switch ISP, <span>you only need to unplug and replug the Internet Cube from the old box to the new one and your server will be available again</span> without the need of any configuration;</li>
+              <li>if you move or switch off your electricity to go on vacation, <span>you can plug your Cube in on any internet connection</span> without having to do any configuration;</li>
+              <li><span>you can even connect your portable server to an especially restricted access,</span>, like a 3/4G connection and continue to offer your services to the whole world.</li>
+            </ul>
+          </li>
+          <li>
+            <strong>Bonus:</strong>
+            <ul>
+              <li>with a maximum consumption of only 2W with the wifi activated, the Cube can be powered <span>via a battery or a small solar Panel</span> (a 12€ battery cell can last 13h);</li>
```
Comment (keomabrun) [inline en.html:78]: wifi, WiFi or Wi-Fi ?

File: en.html [hunk 2]
```diff
@@ -32,63 +32,60 @@ <h1><img src="img/cube.png" alt="The Internet Cube" ></h1>
...
+              <li>the box is <span>open-hardware and all the software included is free software</span>;</li>
```
Comment (keomabrun) [inline en.html:79]: link to the open-hardware concept ?

File: en.html [hunk 3]
```diff
@@ -32,63 +32,60 @@ <h1><img src="img/cube.png" alt="The Internet Cube" ></h1>
...
+              <li>the Internet Cube can also allow you to <span>share files easily</span> during an event (this is what we call a <em>PirateBox</em>) ;</li>
+              <li>you can also use it <span>to provide internet access at a public event</span>, when using someone elses connection to free it's owner from liabilities.</li>
```
Comment (keomabrun) [inline en.html:81]: Internet or internet ?

File: index.html [hunk 4]
```diff
@@ -32,59 +32,60 @@ <h1><img src="img/brique.png" alt="La Brique Internet" ></h1>
...
+
+      <p>1. Elle <strong>nettoie votre accès à Internet</strong>. Pour ce faire, elle diffuse un réseau WiFi. En passant par celui-ci, tout votre traffic Internet passe à travers <a href="https://fr.wikipedia.org/wiki/R%C3%A9seau_priv%C3%A9_virtuel">un tunnel chiffré (VPN)</a>. Ainsi, votre fournisseur d'accès à Internet (FAI) ne peut plus espionner, filtrer ou brider votre accès à Internet. Dans le futur, la brique pourra intégrer des fonctions supplémentaires telles que le blocage automatique des publicités et trackers, ou d'autres mesures permettant de protéger votre vie privée.</p>
+
+      <p>2. Elle permet d'<strong>avoir son propre bout d'Internet</strong>. C'est ce qu'on apelle l'auto-hébergement : le fait de disposer de son propre serveur qui héberge des services comme un blog, une boîte mail, de la synchronisation de fichiers, de contacts, de calendrier, etc.. Cette pratique est simplifiée par le logiciel <a href="https://yunohost.org">YunoHost</a>. La Brique Internet permet donc de s'affranchir des services centralisés proposés par <a href="https://fr.wikipedia.org/wiki/GAFAM">les GAFAMs</a> et ainsi de reprendre le contrôle de ses données.</p>
```
Comment (keomabrun) [inline index.html]: appelle

File: index.html [hunk 5]
```diff
@@ -32,59 +32,60 @@ <h1><img src="img/brique.png" alt="La Brique Internet" ></h1>
...
+        La Brique Internet permet deux principaux usages :
+
+        <ol>
+          <li>
+            <strong>Nettoyer son accès à Internet :</strong>
+            <ul>
+              <li><span>la Brique se branche simplement sur la box</span> de votre fournisseur d'accès à Internet (FAI)... c'est tout ;</li>
+              <li><span>elle diffuse un second réseau WiFi</span>, que vous pouvez désormais utiliser pour accéder à Internet ;</li>
+              <li>en passant par ce WiFi plutôt que celui de votre FAI, ce dernier <span>ne sera plus capable d'espionner, ni filtrer, ni brider votre accès</span> à Internet ;</li>
+              <li>en bonus, <span>vos ordinateurs auront tous de l'IPv6 natif</span>, sans ne rien avoir eu à configurer ;</li>
```
Comment (keomabrun) [inline index.html:58]: si le fournisseur de VPN fourni de l'IPv6...

**Review by keomabrun — CHANGES_REQUESTED** [review]

**Comment by alexAubin** [comment]
@keomabrun : well uh, basically all those were from the previous text (which I just moved) so @.@

**Comment by alexAubin** [comment]
𝗌𝗆𝗈𝗅 𝖻𝗎𝗆𝗉

**Comment by alexAubin** [comment]
Marçi <3

**Pull request merged**